cmd/influxd/inspect/dump_tsm/dump_tsm.go

```diff
@@ -178,7 +187,10 @@ func dumpIndex(cmd *cobra.Command, args args, info dumpIndexParams) {
 			split := strings.Split(string(key), "#!~#")
 
 			measurement := split[0]
-			field := split[1]
+			var field string
+			if len(split) > 1 {
```

Comment:
When will `split` have len=1?

Reply:
This occurred in my test case, where I simply wrote a `tsm1.Value()` with just a simple name and measurement and nothing else